<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagevalWidget</class>
 <widget class="QWidget" name="ImagevalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ckeck image parameters</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-properties">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="extLabel">
     <property name="text">
      <string>Image formats</string>
     </property>
     <property name="buddy">
      <cstring>extEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dpiLabel">
     <property name="text">
      <string>Requested resolution</string>
     </property>
     <property name="buddy">
      <cstring>dpiSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="dpiSpin">
     <property name="specialValueText">
      <string comment="resolution">Any</string>
     </property>
     <property name="suffix">
      <string notr="true"> DPI</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="minqualLabel">
     <property name="text">
      <string>Minimum quality</string>
     </property>
     <property name="buddy">
      <cstring>minqualSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="minqualSpin">
     <property name="suffix">
      <string notr="true">%</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="levelLabel">
     <property name="text">
      <string>Level</string>
     </property>
     <property name="buddy">
      <cstring>levelSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="levelSpin">
     <property name="specialValueText">
      <string comment="level">Any</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="extEdit">
     <property name="placeholderText">
      <string>Comma separated extensions</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="maxqualLabel">
     <property name="text">
      <string>Maximum quality</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="maxqualSpin">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>80</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>extEdit</tabstop>
  <tabstop>dpiSpin</tabstop>
  <tabstop>minqualSpin</tabstop>
  <tabstop>levelSpin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
